utils.py

```diff
@@ -76,13 +109,40 @@ def dataset_from_folder(folder, img_ext='.png', mask_ext='.trn'):
         yield load_image_for_dataset(file)
 
 if __name__ == '__main__':
-    for datum in dataset_from_folder('./plzen'):
+    data, data_mask = load_dataset('./plzen')
+    eigenvalues, pc = calc_PCA(data)
+    jitter_data = ra.add_color_noise(data, eigenvalues, pc)
+    pca_data = np.clip(jitter_data * 255, 0, 255).astype('uint8')
+
+    flipped_data, flipped_mask = ra.flip_batch(data, data_mask)
```

Comment:
@Foxiy all of these things could probably happen as part of `dataset_from_folder`. The only problem I see there is that `flip_batch` works on batches (obviously) while `dataset_from_folder` reads (and yields out) the data in a sequential manner.

What do you think?